operations_on_datastructures/linked_list_right_rotation.cpp

```diff
@@ -0,0 +1,215 @@
+/**
+ * @file
+ * @brief Rotate a linked list 'n' number of times.
```

Comment:
Provide a Wikipedia link in Markdown format (if available/possible).
If there's no Wikipedia link, add another web source for algorithm explanation. 🙂